nuxeo-core/nuxeo-core-test/src/test/resources/OSGI-INF/test-document-blob-coldstorage-contribs.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.coldstorage.test">
+
+  <extension target="org.nuxeo.ecm.core.api.thumbnail.ThumbnailService"
+             point="thumbnailFactory">
+    <thumbnailFactory name="fakeThumbnailFactory"
+                      factoryClass="org.nuxeo.ecm.core.FakeThumbnailFactory" />
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.blob.BlobManager" point="configuration">
+    <blobprovider name="coldstorage">
+      <class>org.nuxeo.ecm.core.blob.InMemoryBlobProvider</class>
+    </blobprovider>
+    <blobprovider name="dummy">
```

Comment:
Since this contribution already exists in `test-dummy-blob-provider.xml`, maybe we could deploy this contribution in the test to avoid duplication here?

And probably, at some point, we'll have a `ColdStorageFeature` deploying `test-dummy-blob-provider.xml`, `test-document-blob-coldstorage-contribs.xml`, etc.